docs/_source/getting_started/installation/configurations/user_management.md

```diff
@@ -42,99 +42,100 @@ rg.init(workspace="my_shared_workspace")
-
 ## Default user
 
-By default, if you don't configure a `users.yml` file, your Argilla instance is pre-configured with the following default user:
+By default, if the Argilla instance has no users, the following default admin user will be configured:
 
 - username: `argilla`
 - password: `1234`
 - api_key: `argilla.apikey`
 
-for security reasons we recommend changing at least the password and API key.
+For security reasons, we recommend changing at least the password and the API key. You can configure
+the default user as follows:
+
+```bash
+python -m argilla.tasks.users.create_default --password newpassword --api-key new-api-key
+```
+
 
 :::{note}
 To connect to an old Argilla server using client `>=1.3.0`, you should specify the default user API key `rubrix.apikey`.
 Otherwise, connections will fail with an Unauthorized server error.
 :::
 
-### Override default API key
+## Add new users and workspaces
+
+The above user management model is configured using the Argilla tasks, which server maintainers can define before launching an Argilla instance.
 
-To override the default API key you can set the following environment variable before launching the server:
+### Prepare the database
+First of all, you need to make sure that database tables and models are up-to-date. This task must be launched when a new version of Argilla is installed.
 
 ```bash
-export ARGILLA_LOCAL_AUTH_DEFAULT_APIKEY=new-apikey
+python -m argilla.tasks.database.migrate
 ```
 
+:::{note}
+It is important to launch this task prior to any other database action.
+:::
 
-### Override default password
-
-To override the password, you must set an environment variable that contains an already hashed password.
-You can use `htpasswd` to generate a hashed password:
 
+### Creating an admin user
 ```bash
-htpasswd -nbB "" my-new-password
-:$apr1$n0C4S20a$noG.3yWxH1OIKfFITgzl30
+python -m argilla.tasks.users.create --role admin
 ```
 
-Afterwards, set the environment variable omitting the first `:` character (in our case `$apr1$n0C4...`):
-
+### Creating an annotator user
 ```bash
-export ARGILLA_LOCAL_AUTH_DEFAULT_PASSWORD="<generated_user_password>"
+python -m argilla.tasks.users.create --role annotator
 ```
 
-Alternatively, you can also generate the hash using passlib's CryptContext: E.g., by running the following in a python console:
+### HOWTO Create a new workspace an assign it to an existing annotator
+
+#### 1. Setup a http client with an admin auth header
 
 ```python
-from passlib.context import CryptContext
-print(CryptContext(schemes=["bcrypt"], deprecated="auto").hash('password'))
+auth_headers = {"X-Argilla-API-Key": "argilla.apikey"}
+http = httpx.Client(base_url="http://localhost:6900", headers=auth_headers)
 ```
 
-## Add new users and workspaces
+#### 2. Call the create workspace endpoint
+
+```python
+workspace = http.post("/api/workspaces", json={"name": "new-workspace"}).json()
+workspace
```

Comment:
can you show the output of this variable?

Reply:
done